htmd/parameterization/esp.py

```diff
@@ -52,9 +52,15 @@ class ESP:
     Load water molecule
     >>> import os
     >>> from htmd.home import home
-    >>> from htmd.parameterization.ffmolecule import FFMolecule, FFTypeMethod
+    >>> from htmd.parameterization.fftype import fftype, FFTypeMethod
+    >>> from htmd.parameterization.util import getEquivalentsAndDihedrals, canonicalizeAtomNames
+    >>> from htmd.molecule.molecule import Molecule
     >>> molFile = os.path.join(home('test-qm'), 'H2O.mol2')
-    >>> mol = FFMolecule(molFile, method=FFTypeMethod.GAFF2)
+    >>> mol = Molecule(molFile)
+    >>> mol = canonicalizeAtomNames(mol)
+    >>> parameters, mol = fftype(mol, method=FFTypeMethod.GAFF2)
```

Comment:
No atom renaming and atom typing needed here

Reply:
Thanks. Removed